data/json/items/ranged.json

```diff
@@ -5428,5 +5428,31 @@
     "built_in_mods": ["m203", "rifle_scope", "muzzle_brake"],
     "default_mods": ["pistol_grip", "rail_laser_sight"],
     "valid_mod_locations": [[ "accessories", 10 ],[ "barrel", 10 ],[ "bore", 10 ],[ "conversion", 10 ],[ "grip", 10 ],[ "magazine", 10 ],[ "mechanism", 10 ],[ "muzzle", 10 ],[ "rail", 10 ],[ "sights", 10 ],[ "stock", 10 ],[ "underbarrel", 10 ]]
+  },
+  {
+    "id": "chemical_thrower",
+    "type": "GUN",
+    "symbol": "(",
+    "color": "dark_gray",
+    "name": "chemical thrower",
+    "description": "A bulky, self-made chemical sprayer with an internal conversion mechanism that allows all sorts of chemicals to be dispersed to your surroundings.",
+    "price": 15000,
+    "material": ["steel", "plastic"],
+    "flags": ["FIRE_100", "NEVER_JAMS"],
```

Comment:
Fire 100 is quite a lot. With flamethrowers you can acquire the ammo easily by siphoning it from cars. Here you've got to craft every shot.

Fire 20 or 50 should be fine, unless you intend to make the gas very powerful.